seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/connector-jdbc-e2e-common/src/test/java/org/apache/seatunnel/connectors/seatunnel/jdbc/AbstractJdbcIT.java

```diff
@@ -112,7 +112,7 @@ public abstract class AbstractJdbcIT extends TestSuiteBase implements TestResour
 
     abstract JdbcCase getJdbcCase();
 
-    abstract void compareResult(String executeKey) throws SQLException, IOException;
+    void checkResult(String executeKey, TestContainer container, Container.ExecResult execResult) {}
```

Comment:
add more parameter to help check log. 
and delete the abstract keyword, the implement class only need implement it when necessary.